heroku.yml

```diff
@@ -20,5 +20,5 @@ run:
     image: django
   worker:
     command:
-      - celery -A gpt_playground worker -l INFO --beat --concurrency 2
+      - celery -A gpt_playground worker -l INFO --beat --concurrency 2 --scheduler django_celery_beat.schedulers:DatabaseScheduler
```

Comment:
FYI you can use the `CELERY_BEAT_SCHEDULER` setting to set this globally which will prevent having to set it in each place

Reply:
Ah thanks!